### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by williamscch**
This project lays foundations for my Bookstore website. I created the React part of the app, and also set up routing using React Router.
### Implementations: 

- React Router.
- React components.

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({ book }) => {
```
Comment (FrederickMih) [inline src/components/Book.js:4]: - [ ] Hi @williamscch, Based on the requirements of this project, the button for removing books should be present in this component. This button shouldn't remove the book yet. This is just a placeholder. Kindly implement this requirement and move a step toward approval.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @williamscch  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



# To Highlights

- GitHub workflow has been followed :+1: 
- Pull Request has a title and summary :heavy_check_mark: 
- No Linter error



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.js (hunk 1 above)
Comment (williamscch) [reply src/components/Book.js:4]: Hey, yes that's true thanks for remembering me.

**Review by FrederickMih — APPROVED** [review]
Hi @williamscch , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**